tests/phpunit/tests/theme/wpTheme.php

```diff
@@ -375,6 +375,33 @@ public function test_get_files_nonexistent_theme() {
 		$this->assertEmpty( $files );
 	}
 
+	/**
+	 * Test wp_customize_url with existing query args.
+	 *
+	 * @ticket 63632
+	 */
+	public function test_wp_customize_url_with_existing_query_args() {
+		$clean_admin_url = admin_url( 'customize.php' );
+
+		// Ensure the existing query arg is present in the URL.
+		add_filter(
+			'admin_url',
+			static function ( $url ) {
+				return add_query_arg( 'existing_arg', 'value', $url );
+			}
+		);
+		$this->assertSame( esc_url( $clean_admin_url . '?existing_arg=value&theme=theme' ), wp_customize_url( 'theme' ) );
```

Comment:
Let's use a theme name that isn't confused with the query var key:

```suggestion
		$this->assertSame( esc_url( $clean_admin_url . '?existing_arg=value&theme=foo' ), wp_customize_url( 'foo' ) );
```